packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -240,17 +274,17 @@ export const DatepickerCalendar: ComponentWithAs<'div', DatepickerCalendarProps>
             selected: day.isSelected,
             disabled: !day.isInBounds,
             quiet: !day.isInMonth,
-            isToday: compareDates(day.originalDate, today ?? new Date()),
+            today: compareDates(day.originalDate, props.today ?? new Date()),
             ref: compareDates(gridNavigatedDate, day.originalDate) ? focusDateRef : null,
           }),
         overrideProps: (predefinedProps: DatepickerCalendarCellProps): DatepickerCalendarCellProps => ({
           onClick: e => {
-            onDateChange(e, { ...predefinedProps, value: day });
+            onDateChange(e, { ...props, value: day });
             _.invoke(predefinedProps, 'onClick', e, { ...predefinedProps, value: day });
```

Comment:
```suggestion
            _.invoke(predefinedProps, 'onClick', e, { ...props, value: day });
```

So it matches line 287?
Or should that line be changed instead? :)

Reply:
Yep, you are right. I have it in one of t[he other PRs](https://github.com/rymeskar/fluentui/pull/4/files), will also bring here :)